### PR TITLE
packages: Update to release 3.0.3

### LINKED ISSUE #131: update SCRIPTSDIR path in runtime package
https://github.com/clearcontainers/runtime/pull/664 fix  scripts installation, after being merged change  SCRIPTS_DIR to `/usr/bin/`

### LINKED ISSUE #148: Release 3.0.3
Update the following packages to the 3.0.3 release:
- cc-runtime
- cc-shim
- cc-proxy

### PR DESCRIPTION
**Pull request by erick0z**
Update packages to match CC release 3.0.3

- runtime 3.0.3
- proxy 3.0.3
- shim 3.0.3
- linux-container 4.9.54-78
- clear-containers-image 18220

Fixes #148

Since https://github.com/clearcontainers/runtime/pull/664 is now
included in this release, this patch also includes the runtime's
makefile fix.
Fixes #131

### CODE REVIEW

**Comment by jodh-intel** [comment]
lgtm

File: runtime/cc-runtime.spec-template [hunk 1]
```diff
@@ -105,7 +105,6 @@ make install \
    BASH_COMPLETIONSDIR=%{buildroot}/usr/share/bash-completion/completions/cc-runtime \
    DESTTARGET=%{buildroot}/usr/bin/cc-runtime \
    DESTCONFIG=%{buildroot}/etc/clear-containers/configuration.toml \
-   SCRIPTS_DIR=%{buildroot}/usr/bin/cc-collect-data.sh
```
Comment (gorozco1) [inline runtime/cc-runtime.spec-template]: If you are deleting this line you need to remove the \ from the line above.

File: runtime/debian.rules-template [hunk 2]
```diff
@@ -41,5 +41,4 @@ override_dh_auto_install:
 		BASH_COMPLETIONSDIR=$(shell pwd)/debian/cc-runtime/usr/share/bash-completion/completions/cc-runtime \
 		DESTTARGET=$(shell pwd)/debian/cc-runtime/usr/bin/cc-runtime \
 		DESTCONFIG=$(shell pwd)/debian/cc-runtime/etc/clear-containers/configuration.toml \
-		SCRIPTS_DIR=$(shell pwd)/debian/cc-runtime/usr/bin/cc-collect-data.sh
```
Comment (gorozco1) [inline runtime/debian.rules-template]: If you are deleting this line you need to remove the \ from the line above.

**Comment by erick0z** [comment]
@gorozco1 done

**Review by gorozco1 — APPROVED** [review]

**Pull request merged**